Rakefile

```diff
@@ -1,6 +1,7 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle12sp1
+yast_submit = ENV["YAST_SUBMIT"] || :sle12sp1
+Yast::Tasks.submit_to(yast_submit.to_sym)
```

Comment:
Well, removing the line would change the default target which would be a very bad side effect. So far you always use `rake osb:build` for running a testing build regardless the branch you currently use.

After removing the line you would need use `YAST_SUBMIT=sle12sp1 rake osc:build` in the *SLE-12-SP1* branch and the simpler `rake osc:build` for *master* otherwise you would build the SP1 code against Factory. That would be really confusing...